packages/@okta/vuepress-theme-prose/assets/css/components/_sidebar.scss

```diff
@@ -1,118 +1,52 @@
 .landing-navigation {
-  padding-left: 1.5rem;
-  padding-right: 2rem;
-  flex: 0 0 auto;
-  border-right: 1px solid cv("gray", "100");
-  color: cv("action");
-  position: sticky;
-  overflow-y: auto;
-  top: 155px;
-
-  @include media(">=tablet") {
-    top: 150px;
-  }
-
-  .children-active {
-    font-weight: bold;
-  }
-
-  .blank-divider {
-    color: #6e6f78;
-    cursor: default;
-    margin-top: 12px;
-  }
-
-  .router-link-active {
-    font-weight: bold;
-    border-left: 0.2rem solid cv("gray", "900");
-    margin-left: -1.2rem;
-    padding-left: 1rem;
-    padding-right: 0.65rem;
-    color: cv("gray", "900");
-  }
-
-  .subnav > .link-wrap > .router-link-exact-active {
-    border-left: none;
-    padding-left: 1.2rem;
-    color: inherit;
-  }
-
-  .overview > .router-link-active {
-    font-weight: normal;
-  }
-
-  .overview > .router-link-exact-active {
-    font-weight: bold;
-  }
-
-  ul,
-  li {
-    list-style: none;
-    padding: 0;
-    margin: 0;
-  }
-
-  li {
-    margin-bottom: 0.5rem;
-
-    &.subnav.open > .link-wrap > a {
-      padding-bottom: 0.4rem;
-    }
-
-    .link-wrap {
-      display: flex;
-
-      .item-collapsable {
-        display: flex;
-        align-items: center;
-
-        svg {
-          flex-shrink: 0;
-          margin-right: 0.5em;
-          fill: currentColor;
-          height: 1em;
-          margin-left: -1.2rem;
-          &.open {
-            margin-top: -0.5rem;
-          }
+  width: 100%;
+  padding: 3rem 0;
+  background: #16272f;
```

Comment:
should it be set for `tree-nav`?

Reply:
Tree-nav is container, inner part should be adjusted to the width of container.